### ISSUE TITLE
"Pause" doesn't stop UDP messages

### ISSUE THREAD

**Issue by harriv** [djeedjay/DebugViewPP#235]
When pause is enabled, UDP messages are still collected and displayed.

**Comment by janwilmans** [comment]
good point, it is actually a symptom of a more fundamental problem, I will investigate.
